### ISSUE TITLE
Fix a bit upward position with newly trained checkpoints in simple-romp

### ISSUE THREAD

**Issue by guoees** [Arthur151/ROMP#242]
Hello author, Excuse me, I would like to ask what is the approximate final loss function. I changed the batchsize to 16, and the loss function is bigger after it runs.
Epoch: [119][1593/1653] Time 0.35 RUN 0.77 Lr 5.000000000000001e-07 Loss 220.56 | Losses {'reg': 167.9, 'det': 52.66, 'CenterMap': 52.66, 'P_KP2D': 42.3, 'MPJPE': 29.3, 'PAMPJPE': 37.73, 'Pose': 56.1, 'Shape': 0.14, 'Prior': 2.32}
Epoch: [119][1643/1653] Time 0.34 RUN 0.77 Lr 5.000000000000001e-07 Loss 219.14 | Losses {'reg': 170.75, 'det': 48.4, 'CenterMap': 48.4, 'P_KP2D': 44.36, 'MPJPE': 29.71, 'PAMPJPE': 37.84, 'Pose': 56.38, 'Shape': 0.14, 'Prior': 2.33}

**Comment by Arthur151** [comment]
How are the evaluation results? The loss of CenterMap(for detection) is a bit high. The rest are fine.Get Outlook for Android

**Comment by guoees** [comment]
Thank you very much for your reply, with simple_romp test, the human body position is a bit upward.
My outlook account is: guoees@outlook

**Comment by Arthur151** [comment]
Hi, there,
This bug is caused by some different configuration during training. 
With your trained model, please set `smpl_mesh_root_align=True` during inference. 

To use simple-romp, I just commited a new version to support root_align.
Please git clone the latest repo. Then re-install simple-romp via 
```
pip uninstall simple-romp
cd ROMP/simple-romp
python setup.py install
```
Then please run simple-romp with `--root_align=True` with your checkpoints.

Thanks for the bug report!

**Comment by Arthur151** [comment]
Hi, there,
This bug is caused by some different configuration during training. 
With your trained model, please set `smpl_mesh_root_align=True` during inference. 

To use simple-romp, I just commited a new version to support root_align.
Please git clone the latest repo. Then re-install simple-romp via 
```
pip uninstall simple-romp
cd ROMP/simple-romp
python setup.py install
```
Then please run simple-romp with `--root_align=True` with your checkpoints.
For instance, `romp -m image -i demo.jpg --root_align=True --show`

Thanks for the bug report!

**Comment by guoees** [comment]
Thanks a lot for your help
![YB7P}~@8 JB PYHS{_EX{DD](https://user-images.githubusercontent.com/96688784/169035127-5ca10ac1-6fdb-4f51-9dfb-09f67d735a68.png)
.
show in active_cionfigs
smpl_mesh_root_align: true.
![21~(94WVO }N 4C}@5PN9UV](https://user-images.githubusercontent.com/96688784/169035872-da290461-c4df-4f16-a6b0-c6cfcc0ae87a.png)


I used the latest simple_romp and I got a new error.
![image](https://user-images.githubusercontent.com/96688784/169036529-3e3b78f7-ff5e-4dfa-9ce8-968ed43c6996.png)
I try to modify the default value. The returned result has not changed, the position is still a little up

**Comment by Arthur151** [comment]
Hi, 

1. The config smpl_mesh_root_align is for the old ROMP, not simple-romp.

2.Caution: Maybe you need to uninstall the previous simple-romp via
```
pip uninstall simple-romp
```
Then you can upgrade your simple-romp via 
```
pip install --upgrade simple-romp
```
3. If you want your modification works, please run
```
cd ROMP/simple_romp
python setup.py install

4. It would be much better if you can show me your test image results!
```

**Comment by Arthur151** [comment]
Glad to hear that your problem is well handled.
